products/dialogflow/terraform.yaml

```diff
@@ -42,7 +42,42 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       tier: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
-
+  Intent: !ruby/object:Overrides::Terraform::ResourceOverride
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dialogflow_intent_basic"
+        primary_resource_id: "basic_intent"
+        # moved to handwritten tests to create a new project per test
+        skip_test: true
+        vars:
+          intent_name: "basic-intent"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dialogflow_intent_full"
+        primary_resource_id: "full_intent"
```

Comment:
I think it's reasonable to have at least one of these examples be a test similar to testAccDialogflowIntent. Keeping the examples as generated tests ensures that our examples are always up to date and free from syntax mistakes. Also having the full set of dependencies in the example (project, IAM, service_account) is a helpful resource for people reading the docs.